module/VuFind/tests/integration-tests/src/VuFindTest/Mink/AdvancedSearchTest.php

```diff
@@ -190,6 +190,40 @@ public function testAdvancedSearch()
         $this->assertEquals(0, count($multiSel->getValue()));
     }
 
+    /**
+     * Test that the advanced search form works correctly with a NOT group
+     *
+     * @return void
+     */
+    public function testAdvancedSearchWithNotOperator()
+    {
+        $session = $this->getMinkSession();
+        $page = $this->goToAdvancedSearch($session);
+
+        // Add a group
+        $session->executeScript("addGroup()");
```

Comment:
I think it would be useful to also test a case where we only have a single NOT group.